safedelete/signals.py

```diff
@@ -0,0 +1,12 @@
+import django
+try:
+    from django.db.models.signals import ModelSignal as Signal
+except ImportError:
+    from django.dispatch import Signal
+
+if django.VERSION[1] <= 5:
```

Comment:
You need to use StrictVersion instead in order not to break at the 2.0 release (http://stackoverflow.com/a/11887885).